url-shortener/android/app/src/androidTest/java/com/nubank/urlshortener/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.nubank.urlshortener
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
I supose we can remove it

Reply:
I recommend we keep this file because it will avoid the testing folder being removed and can help the candidates when creating a new test.